src-php/Navigation/Pagination/Pagination.php

```diff
@@ -0,0 +1,165 @@
+    }
+
+
+    public function render(): string
+    {
+        $prevPageNumber = ($this->pageNumberRequested - 1) <= 0 ? 1 : $this->pageNumberRequested - 1;
+        $nextPageNumber = ($this->pageNumberRequested + 1) >= $this->totalPageCount
+            ? $this->totalPageCount
+            : $this->pageNumberRequested + 1;
+
+        return '<hr><div class="main">'
+            . '<div class="displayCounts">' . $this->renderDisplayCounts() . '</div>'
+            . '<div class="center">'
+            . $this->renderSvgAnchorTag('arrow-first', 1)
+            . $this->renderSvgAnchorTag('arrow-back', $prevPageNumber)
+            . '<div class="pagination">' . $this->renderPageAnchorTags() . '</div>'
+            . $this->renderSvgAnchorTag('arrow-next', $nextPageNumber)
+            . $this->renderSvgAnchorTag('arrow-last', $this->totalPageCount)
+            . '</div></div>';
+    }
+
+
+    public static function renderStyles(): string
+    {
+        $styles = file_get_contents(__DIR__ . '/css/Pagination.css');
+
+        if ((bool)$styles) {
+            return (string)$styles;
+        }
+
+        return self::BLANK;
+    }
+
+
+    private function renderDisplayCounts(): string
+    {
+        $pageSize = (int)ceil($this->totalItemCount / $this->totalPageCount);
+        $lastItemCalculated = $this->pageNumberRequested * $pageSize;
+        $lastItemDisplayed = $lastItemCalculated > $this->totalItemCount ? $this->totalItemCount : $lastItemCalculated;
+        $firstItemDisplayed = $lastItemCalculated - $pageSize + 1;
+
+        return sprintf(
+            'Displaying %d - %d of %d',
+            $firstItemDisplayed,
+            $lastItemDisplayed,
+            $this->totalItemCount
+        );
+    }
+
+
+    private function renderPageAnchorTags(): string
+    {
+        if (($this->pageNumberRequested - 2) < 1) {
+            $firstPage = self::FIRST_PAGE;
+            $lastPage = self::MAX_PAGES_TO_DISPLAY;
+        } else if ($this->pageNumberRequested + 2 > $this->totalPageCount) {
+            $firstPage = $this->totalPageCount - (self::MAX_PAGES_TO_DISPLAY - 1);
+            $lastPage = $this->totalPageCount;
+        } else {
+            $firstPage = $this->pageNumberRequested - 2;
+            $lastPage = $this->pageNumberRequested + 2;
+        }
+
+        $anchors = '';
+        for ($pageNumber = $firstPage; $pageNumber <= $lastPage; $pageNumber++) {
+            $anchors .= $this->renderPageAnchorTag($pageNumber);
+        }
+        
+        return $anchors;
+    }
+
+    
+    private function renderPageAnchorTag(int $pageNumber): string
+    {
+        if ($pageNumber === $this->pageNumberRequested) {
+            $class = self::CURRENT_PAGE_CLASS;
+            $href = self::BLANK;
+        } else {
+            $class = self::BLANK;
+            $href = sprintf('href="%s/%d"', $this->pageRoutePath, $pageNumber);
+        }
+
+        return sprintf('<a %s%s>%d</a>', $class, $href, $pageNumber);
```

Comment:
[O] We like to use early return instead of else https://github.com/BrandEmbassy/developers-manifest/issues/365 & https://github.com/BrandEmbassy/developers-manifest/issues/364 - I will mention it in the manifesto as well..

```suggestion
        if ($pageNumber === $this->pageNumberRequested) {
            return sprintf('<a class="current">%d</a>', $pageNumber);
        }
        
        return sprintf('<a href="%s/%d">%d</a>', $this->pageRoutePath, pageNumber, $pageNumber);
        

        return sprintf('<a %s%s>%d</a>', $class, $href, $pageNumber);
```